actuators/Magnetorquer.cpp

```diff
@@ -2,53 +2,64 @@
 
 #include "math/EigenWrapper.h"
 #include <cmath>
+#include <utility>
+#include <iostream>
 
-
-Magnetorquer::Magnetorquer(int N_MTBs, double maxVolt, double coilsPerLayer, double layers, double traceThickness,
-                           double traceWidth, double gapWidth, double maxPower, double maxCurrentRating,
-                           MatrixXd mtb_orientation) 
+Magnetorquer::Magnetorquer(int N_MTBs, VectorXd maxVolt, VectorXd coilsPerLayer, VectorXd layers, VectorXd traceThickness,
+                           VectorXd pcbSideMax, VectorXd traceWidth, VectorXd gapWidth, VectorXd maxPower, VectorXd maxCurrentRating,
+                           MatrixXd mtb_orientation) : num_MTBs(N_MTBs), max_voltage(std::move(maxVolt)), N(std::move(coilsPerLayer)), pcb_layers(std::move(layers)), 
```

Comment:
I get that we want to account for variations in MTBs in Monte-Carlo, but do we really need to perturb all these quantities? Won't simply perturbing Coil resistance do? I.e., 
$$R_{mtb}^{i=1:N_{mtb}} =R_{nominal}+\Delta R_{i}^{i=1:N_{mtb}}$$

where $R_{nominal}$ is a double and the other two are vectors

Reply:
yes, perturbing R is enough. We can change interfaces here

Reply:
Are we making this change in this PR?

Reply:
hmm I dont think it should be blocking us at this point. We can change this later, lets worry about having a single integrated and validated sim first